util/util.py

```diff
@@ -193,11 +194,10 @@ def build_from_dict(cls, name, contents):
         return cls(name, modules, tests)
 
 
-class Build(BaseBuild):
+class BuildOpt(BaseBuild):
```

Comment:
Is this class used anywhere? I can see you did not add any tests for it.

I am thinking of changing so that BaseBuild subclasses are the only datastructures used within DiffEng. Had I used that approach from the beginning, we would be able to more easily change the internal data structure layout as you did without changing any of the code that depended on it.